### PR TITLE
Reverse conditional_derivative_decorator logic

### PR DESCRIPTION
**Pull request by kirkkwang**
ref #398 

This commit also address its spec however one of them do not pass and
will get addressed in a separated commit.

spec/extensions/hyrax/conditional_derivative_decorator_spec.rb:15 gives a false positve

spec/extensions/hyrax/conditional_derivative_decorator_spec.rb:21 does not pass

### I created an attachment w intermediate file type and attached a file set to it. The file set should render a thumbnail. ✅ 

image

image

### I created an attachment that's not an intermediate file type and attached a file set to it. The file set should NOT render a thumbnail. ✅ 

image

image

### CODE REVIEW

**Comment by laritakr** [comment]
I'm confused about how this file relates to the derivative custom_derivative_service.

**Comment by kirkkwang** [comment]
flip this logic too
https://github.com/scientist-softserv/utk-hyku/blob/5a8b7a6d7c95ff1dfcdf81533a84bc6cda08aba8/app/services/custom_derivative_service.rb#L6

**Comment by ShanaLMoore** [comment]
> flip this logic too
> 
> https://github.com/scientist-softserv/utk-hyku/blob/5a8b7a6d7c95ff1dfcdf81533a84bc6cda08aba8/app/services/custom_derivative_service.rb#L6

✅ DONE

File: app/services/custom_derivative_service.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # @api public
 module CustomDerivativeService
   def valid?
-    false if file_set.rdf_type&.join&.downcase&.include?("intermediate_file")
+    true if file_set.rdf_type&.join&.downcase&.include?("intermediate_file")
```
Comment (ShanaLMoore) [inline app/services/custom_derivative_service.rb:6]: This doesn't seem to be getting called (yet), however, the logic needs to be updated nonetheless.

**Review by ShanaLMoore — APPROVED** [review]

**Pull request merged**